src/mito2/src/cache.rs

```diff
@@ -270,33 +270,42 @@ impl CacheManagerBuilder {
             Cache::builder()
                 .max_capacity(self.sst_meta_cache_size)
                 .weigher(meta_cache_weight)
-                .eviction_listener(|k, v, _cause| {
+                .eviction_listener(|k, v, cause| {
                     let size = meta_cache_weight(&k, &v);
                     CACHE_BYTES
                         .with_label_values(&[SST_META_TYPE])
                         .sub(size.into());
+                    CACHE_EVICTION
+                        .with_label_values(&[SST_META_TYPE, &format!("{:?}", cause)])
```

Comment:
nitpick: Maybe we can implement a function to match the cause and return a `&'static str` to avoid allocating a string. It also improves the readability of the label value.